packages/block-editor/src/components/block-list/use-multi-selection.js

```diff
@@ -147,7 +147,7 @@ export default function useMultiSelection( ref ) {
 		selectedBlockClientId,
 	] );
 
-	const onSelectionChange = useCallback( () => {
+	const onSelectionChange = useCallback( ( { isFinal } ) => {
```

Comment:
Is `isFinal` meant to represent the end of a selection, corresponding to the `onSelectionEnd` event? Might it be something to name similar then, e.g. `isEnd` or `isSelectionEnd` ?

Reply:
Sounds good